packages/search-item/src/owc-search-item.js

```diff
@@ -0,0 +1,187 @@
+import { LitElement, html, css } from 'lit-element';
+
+import '@github/time-elements';
+
+export class OwcSearchItem extends LitElement {
+  static get properties() {
+    return {
+      name: { type: String },
+      description: { type: String },
+      size: { type: String },
+      sizeGzip: { type: String },
+      version: { type: String },
+      versionTime: { type: String },
+      iframedemo: {},
+      codepen: {},
+      registeredType: { type: String },
+      npmUrl: { type: String },
+      unpkgUrl: { type: String },
+      githubStars: { type: Number },
+      githubUrl: { type: String },
+      bundlephobiaUrl: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.registeredType = '';
+    this.name = '';
+    this.title = '';
+    this.version = '';
+    this.description = '';
+    this.size = 0;
+    this.sizeGzip = 0;
+    this.unpkgUrl = '';
+    this.npmUrl = '';
+    this.githubStars = 0;
+    this.githubUrl = '';
+    this.bundlephobiaUrl = '';
+  }
+
+  renderRegisteredTypes() {
+    switch (this.registeredType) {
+      case 'lit-element@2.x.x':
+        return html`
+          <a href="#"><img src="https://img.shields.io/badge/lib-lit--element--2.x-orange.svg"/></a>
+        `;
+      case 'haunted@4.x.x':
+        return html`
+          <a href="#"><img src="https://img.shields.io/badge/lib-haunted--4.x-blueviolet.svg"/></a>
+        `;
+      default:
+        return html``;
+    }
+  }
+
+  render() {
+    return html`
+      <div id="name">
```

Comment:
```suggestion
      <header id="name">
```

Reply:
consider also wrapping the whole tree in an `<article>`

Reply:
got changed quite a lot 🙈  please revisit if needed 👍